docs/_articles/en/linting.md

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
I think this needs to be added to the sidebar

Reply:
I'm not sure what you mean—linting is already in the sidebar.

Reply:
The sidebar exclusively references the other lint article (docs/_articles/en/lwc/linting.md) https://github.com/forcedotcom/salesforcedx-vscode/blob/f6d905a4d53ab7599ff94cd231c5e5228a046445/docs/_data/sidebar.yml#L141